File: app/src/main/java/com/example/rahmatantravel/UbahKataSandi.java
package com.example.rahmatantravel;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class UbahKataSandi extends AppCompatActivity {

    ImageView backToMenuLainnya;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_ubah_kata_sandi);

        backToMenuLainnya = findViewById(R.id.backToMenuLainnya);
        backToMenuLainnya.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                onBackPressed();
            }
        });
    }
}

File: app/src/main/java/com/example/rahmatantravel/UbahKataSandi.kt
package com.example.rahmatantravel;

import android.os.Bundle
import android.view.View
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView

class UbahKataSandi : AppCompatActivity() {

    private lateinit var backToMenuLainnya: ImageView
    private lateinit var buttonSave : CardView


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_ubah_kata_sandi)

        backToMenuLainnya = findViewById(R.id.backToMenuLainnya)
        backToMenuLainnya.setOnClickListener { onBackPressed() }

        buttonSave = findViewById(R.id.btn_save)
        buttonSave.setOnClickListener { onBackPressed() }
    }
}
